mastering-plone/code.rst

```diff
@@ -6,17 +6,10 @@ You can get the complete code for this training from `GitHub <https://github.com
 The code-package
 ----------------
 
-The package
-
-..  note::
-
-    If you want to do it by hand do the following:
-
-    .. code-block:: bash
-
-        cd src
-        git clone https://github.com/collective/ploneconf.site.git
+The package `ploneconf.site <https://github.com/collective/ploneconf.site>`_ contains the complete code for this training (excluding exercises). It is automatically downloaded from github when you run buildout.
 
+The master branch of that repository holds the code of the final chapter of this training.
+Each chapter that adds code to the package has a tag that can be used to get the code for that chapter.
```

Comment:
Ah, thanks for clarifying that it is a git tag, not a branch... Maybe that needs to be explicitly said. 

This might also be where you say which directory you need to be in when you do the git checkout, i.e., in src/ploneconf.site, and whether you need to run bin/buildout afterwards (from the original buildout directory, restart the instance with `bin/instance fg`, and if you need to uninstall and reinstall the ploneconf.site add-on from the Add-ons control panel for the code changes to take effect.

Reply:
- I agree with Kim on this
- I'm also wondering if we need to add a note mentioning that checking out a tag will make git warn you about detached head state, depending on how git savvy the students are this might useful (to just let them know they can safely ignore the warning)
- let me know, I could make a suggestion for this in a seperate commit / PR

Reply:
Good idea @spereverde ! You could make the change directly to this branch `improve_get_code`